test/SourceKit/DocSupport/doc_internal_closure_label.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: %empty-directory(%t.mod)
+// RUN: %swift -emit-module -o %t.mod/label.swiftmodule %s -parse-as-library -emit-module-doc-path %t.mod/label.swiftdoc
+// RUN: %sourcekitd-test -req=doc-info -module label -- -I %t.mod > %t.response
+// RUN: %diff -u %s.response %t.response
```

Comment:
Could you add `.swiftinterface` test?

Reply:
Updated the test to also test the generated `.swiftinterface`.